lib/core/shell/step.sh

```diff
@@ -102,25 +102,35 @@ BOOTSTRAP_SOURCE_LINE='
 . "$HOME/.ih/augment.sh"
 '
 
-# Create bashrc if it doesn't exist, if it does, append standard template
-function ih::setup::core.shell::private::configure-bashrc() {
-  if [[ ! -e "${HOME}/.bashrc" ]]; then
+function ih::setup::core.shell::private::configure-bash() {
+  local configFile
+
+  # On many Unix-like systems, non-login shells typically source the `.bashrc` file.
```

Comment:
Don't you also need to change `ih::setup::core.shell::test` to test the correct file?

Also, maybe you should just ensure `.bash_profile` sources `.bashrc` if it exists, so that all grounds are covered?

Reply:
Added the .bash_profile check to `ih::setup::core.shell::test`: https://github.com/ConsultingMD/homebrew-ih-public/pull/67/commits/e391a5b3dd0f0e8e7b73263512ebc113cbe31c1d

I was thinking about sourcing .bashrc from .bash_profile (that's how I had it set up last time I used bash), but I was thinking that could backfire if they've got some conflicting stuff in the two files.

But the way I currently have it assumes a fresh Mac install without a `.bashrc`, which wouldn't handle that scenario.

Maybe we do that as a round 2 sort of thing later?

Reply:
> that could backfire if they've got some conflicting stuff in the two files

Lots of things could backfire for existing installations... like this fix not doing anything because someone already has a broken .bashrc in there.

I'm not sure what's the answer here. Do what you think is best.

Reply:
> like this fix not doing anything because someone already has a broken .bashrc in there.

The .bashrc won't exist on a fresh mac install, just the .bash_profile. This handles the case when someone has a fresh mac install and switches to using bash as their default shell, like we saw @includedevan do recently.

Reply:
This script isn't only run on fresh installations. Anyway, I'm not familiar with it that much, nor can I even run it, so...

Reply:
@vanzin I just updated it to `source ~/.bashrc` from `~/.bash_profile`.